DemoApp/Assets/Scripts/HomeScreen.cs

```diff
@@ -7,31 +7,36 @@ public class HomeScreen : MonoBehaviour
     private const string InterstitialAdUnitId = "ENTER_INTERSTITIAL_AD_UNIT_ID_HERE";
     private const string RewardedAdUnitId = "ENTER_REWARD_AD_UNIT_ID_HERE";
     private const string BannerAdUnitId = "ENTER_BANNER_AD_UNIT_ID_HERE";
+    private const string MRecAdUnitId = "ENTER_MREC_AD_UNIT_ID_HERE";
 
     public Button showInterstitialButton;
     public Button showRewardedButton;
     public Button showBannerButton;
+    public Button showMRecButton;
     public Button mediationDebuggerButton;
     public Text interstitialStatusText;
     public Text rewardedStatusText;
 
-    private bool isBannerShowing = false;
+    private bool isBannerShowing;
+    private bool isMRecShowing;
 
     void Start()
     {
         showInterstitialButton.onClick.AddListener(ShowInterstitial);
         showRewardedButton.onClick.AddListener(ShowRewardedAd);
         showBannerButton.onClick.AddListener(ToggleBannerVisibility);
+        showMRecButton.onClick.AddListener(ToggleMRecVisibility);
         mediationDebuggerButton.onClick.AddListener(MaxSdk.ShowMediationDebugger);
 
-        MaxSdkCallbacks.OnSdkInitializedEvent += (MaxSdkBase.SdkConfiguration sdkConfiguration) =>
+        MaxSdkCallbacks.OnSdkInitializedEvent += sdkConfiguration =>
         {
-            // AppLovin SDK is initialized, configure and start loading ads
+            // AppLovin SDK is initialized, configure and start loading ads.

```

Comment:
nit but going forward no need to modify existing comments' style - I usually don't add periods to comments _within_ code but I know some folks like Santosh does - so no need to re-write their stuff since we all have diff styles. For javadocs / header docs I agree we should have complete sentences with periods tho